src/components/FunctionNotation.js

```diff
@@ -26,8 +26,7 @@ class FunctionNotation extends React.Component {
                     <img 
                         src={"https://i.ibb.co/zP2Ytm0/Function-Notation.png"}
                         alt="Function Notation"
-                        width={350}
-                        height={350} />
+                        style={{ width: "30%" }} />
```

Comment:
This is not specific to this image, but adding a minWidth property for all of the images would be better. It helps the image become more visible for smaller devices, as the percentage can make it relatively small. 

Something like this should work: style={{  width: "30%", minWidth: "300px", height: "auto"}}